go/genesis/api/api.go

```diff
@@ -35,6 +40,12 @@ type Document struct {
 	Staking staking.Genesis `json:"staking"`
 	// KeyManager is the key manager genesis state.
 	KeyManager keymanager.Genesis `json:"keymanager"`
+	// Scheduler is the scheduler genesis state.
+	Scheduler scheduler.Genesis `json:"scheduler"`
+	// Beacon is the beacon genesis state.
+	Beacon beacon.Genesis `json:"beacon"`
+	// Consensus is the consensus genesis state.
+	Consensus tendermint.Genesis `json:"consensus"`
```

Comment:
This structure should probably not be defined in `tendermint` but in `go/common/consensus` (to make it consensus-backend agnostic).